test/acceptance.js

```diff
@@ -30,7 +30,7 @@ describe('objects', () => {
     const obj = {
       inspect: () => 1,
     }
-    expect(inspect(obj, { customInspect: true })).to.equal(1)
+    expect(inspect(obj, { customInspect: true })).to.equal('1')
```

Comment:
This is probably a good change because we ideally want loupe to always return a string.